src/PactNet.Native/NativeMockServer.cs

```diff
@@ -8,19 +9,9 @@ namespace PactNet.Native
     /// </summary>
     internal class NativeMockServer : IMockServer
     {
-        /// <summary>
-        /// Static constructor for <see cref="NativeMockServer"/>
-        /// </summary>
-        static NativeMockServer()
-        {
-            // TODO: make this configurable somehow, except it applies once for the entire native lifetime, so dunno
-            Environment.SetEnvironmentVariable("LOG_LEVEL", "DEBUG");
-            Interop.Init("LOG_LEVEL");
```

Comment:
Are the logs handled somewhere else or it's a proactive change?

Reply:
The logs part is in the other PR so that this is a single change just to swap to the new FFI. I'll refactor the other one into this once it's merged, because the other doesn't work on Linux without this change.